examples/notebooks/voltage-regulator/publish_waveforms.ipynb

```diff
@@ -12,48 +12,118 @@
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "5aaa09bb",
+   "id": "72711880",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from ni.datastore import Client, Operator, TestResult, TestStation\n",
+    "from ni.datastore import (\n",
+    "    Client,\n",
+    "    HardwareItem,\n",
+    "    Operator,\n",
+    "    SoftwareItem,\n",
+    "    TestStation,\n",
+    "    Uut,\n",
+    "    UutInstance,\n",
+    ")\n",
+    "\n",
+    "# Constants\n",
+    "ALIAS_JAMES_BOWERY = \"Operator_BoweryJ\"\n",
+    "ALIAS_TEST_STATION_12 = \"TestStation_12\"\n",
+    "ALIAS_UUT_A861_42367 = \"UUT_A861_42367\"\n",
+    "ALIAS_SOFTWARE_ITEM_WINDOWS_10 = \"Windows_10.0.19044\"\n",
+    "ALIAS_SOFTWARE_ITEM_PYTHON_3_12 = \"Python_3.12\"\n",
+    "ALIAS_HARDWARE_ITEM_PXIe_5171 = \"HardwareItem_PXIe-5171\"\n",
     "\n",
     "client = Client()\n",
     "\n",
+    "# Create alias for Operator \"James Bowery\"\n",
     "operator = Operator(operator_name=\"James Bowery\", role=\"Test Architect\")\n",
-    "operator_id = client.create_operator(operator)\n",
+    "client.create_operator(operator)\n",
+    "alias_james_bowery = client.create_alias(ALIAS_JAMES_BOWERY, operator)\n",
     "\n",
+    "# Create alias for Test Station \"TestStation_12\"\n",
     "test_station = TestStation(test_station_name=\"TestStation_12\")\n",
-    "test_station_id = client.create_test_station(test_station)\n",
+    "client.create_test_station(test_station)\n",
+    "alias_test_station_12 = client.create_alias(ALIAS_TEST_STATION_12, test_station)\n",
+    "\n",
+    "# Create alias for UUT instance \"A861-42367\"\n",
+    "uut = Uut(model_name=\"NI-6508\", family=\"Digital\")\n",
+    "uut_id = client.create_uut(uut)\n",
+    "uut_instance = UutInstance(uut_id=uut_id, serial_number=\"A861-42367\")\n",
+    "client.create_uut_instance(uut_instance)\n",
+    "alias_uut_a861_42367 = client.create_alias(ALIAS_UUT_A861_42367, uut_instance)\n",
+    "\n",
+    "# Create aliases for SoftwareItems\n",
+    "software_item = SoftwareItem(product=\"Windows\", version=\"10.0.19044\")\n",
+    "client.create_software_item(software_item)\n",
+    "alias_software_windows_10 = client.create_alias(ALIAS_SOFTWARE_ITEM_WINDOWS_10, software_item)\n",
+    "software_item_2 = SoftwareItem(product=\"Python\", version=\"3.12\")\n",
+    "client.create_software_item(software_item_2)\n",
+    "alias_software_python_3_12 = client.create_alias(ALIAS_SOFTWARE_ITEM_PYTHON_3_12, software_item_2)\n",
+    "\n",
+    "# Create aliases for HardwareItem\n",
+    "scope = HardwareItem(\n",
+    "    manufacturer=\"NI\",\n",
+    "    model=\"PXIe-5171\",\n",
+    "    serial_number=\"1933B4E\",\n",
+    ")\n",
+    "client.create_hardware_item(scope)\n",
+    "alias_scope = client.create_alias(ALIAS_HARDWARE_ITEM_PXIe_5171, scope)\n",
```

Comment:
There is no need to store alias_XYZ variables. In the future we will use the string constant to refer to the alias.

``` python
operator = Operator(operator_name="James Bowery", role="Test Architect")
client.create_operator(operator)
client.create_alias(ALIAS_JAMES_BOWERY, operator)
```